app/views/includes/head.njk

```diff
@@ -14,7 +14,13 @@
   ga('set', 'displayFeaturesTask', null);
   ga('set', 'transport', 'beacon');
 
-  ga('send', 'pageview');
+  var url = new URL(document.location.href);
+  var search_params = new URLSearchParams(url.search);
```

Comment:
You should be able to retrieve searchParams from url itself https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams